macros/CSS_Ref.ejs

```diff
@@ -116,7 +116,7 @@ if (types.some(function(type) { return type === "types"; })) {
 
         var item = {
             urlPath: "",
-            label: type
+            label: "<" + type + ">"
```

Comment:
Similar code to break dependency on https://github.com/mdn/data/pull/81 merging first:

```
        var label = type;
        if (label.charAt(0) != '<') {
            // https://github.com/mdn/data/pull/81 was merged
            label =  "<" + type + ">";
        }
        var item = {
            urlPath: "",
            label: label
        };
```